.github/workflows/ember-assets.yml

```diff
@@ -0,0 +1,24 @@
+name: Ember Asset Sizes
+
+on:
+  pull_request:
+    paths:
+    - 'ui/**'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2-beta
+        with:
+          fetch-depth: 0
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+      - uses: backspace/ember-asset-size-action@edit-comment
```

Comment:
Are you maintaining a fork now or is the long-term plan still to switchover to the upstream action (assuming your changes get merged)?

Reply:
I’d definitely prefer to switch to not have fragmentation, there just hasn’t been any followup. But I’ll keep checking.